task.py

```diff
@@ -612,6 +688,10 @@ def __init__(self, ts: TestSettings):
         self.out_file_yaml = out_file_yaml
         self.pod_name = pod_name
 
+        if self.connection_mode == ConnectionMode.MULTI_NETWORK:
+            self.create_ingress_multi_network_policy(self.port)
+            self.create_egress_multi_network_policy(self.port)
```

Comment:
it's important that the `__init__()` does not already perform such actions that have side effects. That is, because the subclass probably will chain up the `super().__init__(...)` method. Since `ServerTask.__init__()` performs relevant initializations, the subclass probably want to chain up as first thing. That leaves not a nice place how the subclass can still affect what the parent's `__init__()` does.

The simple solution is to move this code to `initialize()`. That's what the method is there for.